keyboards/handwired/yamsek/yamsek.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2022 Johannes Schneider (@JohSchneider) */
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+
+#include "quantum.h"
+
+#ifdef RGBLIGHT_ENABLE
+#    include "rgblight.h"
+#endif
+
+#include <gpio.h>
+// defined in the used mainboard/MCU
+extern const pin_t row_pins_MCU[MATRIX_ROWS / 2];
+extern const pin_t col_pins_MCU[MATRIX_COLS];
+
+
+// This a shortcut to help you visually see your layout.
+// The following is an example using the Planck MIT layout
+// The first section contains all of the arguements
+// The second converts the arguments into a two-dimensional array
+#define LAYOUT( \
+    k00, k01, k02, k03, k04, k05, \
+    k10, k11, k12, k13, k14, k15, \
+    k20, k21, k22, k23, k24, k25, \
+    \
+    k30, k31, k32,     k33, k34, k35, \
+    \
+    \
+    k40, k41, k42, k43, k44, k45,     \
+    k50, k51, k52, k53, k54, k55,     \
+    k60, k61, k62, k63, k64, k65,     \
+                                      \
+    k70, k71, k72,     k73, k74, k75  \
```

Comment:
This should resemble the keyboard layout, and from what is posted above, it looks like this is more accurate for what you want: 

```suggestion
    k40, k41, k42, k43, k44, k45,         k00, k01, k02, k03, k04, k05, \
    k50, k51, k52, k53, k54, k55,         k10, k11, k12, k13, k14, k15, \
    k60, k61, k62, k63, k64, k65,         k20, k21, k22, k23, k24, k25, \
    \
    k70, k71, k72,     k73, k74, k75      k30, k31, k32,     k33, k34, k35, \
    \
    \
                                      
```

Reply:
oh, now i see - looking left&right at other keyboards like dactyl, ...
thanks for the hint!

actually it should be sth. like this then - if this block is to follow the exact physical key-switch layout: 

```suggestion
    k40, k41, k42, k43, k44, k45,                                       k00, k01, k02, k03, k04, k05, \
    k50, k51, k52, k53, k54, k55,                                       k10, k11, k12, k13, k14, k15, \
    k60, k61, k62, k63, k64, k65,                                       k20, k21, k22, k23, k24, k25, \
    \
                                  k72,    k73, k74, k75,    k30, k31, k32,   k33, \
                               k71, k70,                                               k35, k34 \
```

Reply:
I wasn't entirely sure on the boards appearance, but yeah.